### PR TITLE
update runScripts with new return from com1DFAMain

### PR DESCRIPTION
**Pull request by awirb**
* now only 4 return parameters from com1DFAMain - update of runScripts was required 
* update benchmark ini files releaseScenario (move to section INPUT) and relTh (add relThFromShp flag value) values according to new settings

### CODE REVIEW

File: avaframe/com1DFA/com1DFATools.py [hunk 1]
```diff
@@ -228,3 +230,37 @@ def extendProfileBottom(cfg, dem, profile):
...
+        inputDir = pathlib.Path(inputDir)
+
+    for tStep in timeSteps:
+        tFiles = sorted(list(inputDir.glob('*%s_t%.2f*' % (resType, tStep))))
+
+    fieldsList = []
+    for tFile in tFiles:
+        fieldsList.append({resType: IOf.readRaster(str(tFile))['rasterData']})
+
+    return fieldsList
```
Comment (matthiasto) [inline avaframe/com1DFA/com1DFATools.py:266]: there is in com1DFA a readFields function, could they maybe be combined or is there a difference between the two?

**Review by matthiasto — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/com1DFATools.py (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFATools.py]: I merged the two functions

File: avaframe/com1DFA/com1DFA.py [hunk 2]
```diff
@@ -1888,20 +1891,25 @@ def trackParticles(cfgTrackPart, dem, particlesList):
...
+        resType: list
+            list of desired result types
         simName : str
             simulation name
         flagAvaDir: bool
             if True inDir corresponds to an avalanche directory and pickles are
             read from avaDir/Outputs/com1DFA/particles
         comModule: str
             module that computed the particles
+        timeStep: float
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:1910]: How does this have to be specified? Does it have to be precise or is a range considered? -> Provide info here

File: avaframe/com1DFA/particleTools.py [hunk 3]
```diff
@@ -21,20 +21,22 @@
 log = logging.getLogger(__name__)
 
 
-def initialiseParticlesFromFile(cfg, avaDir):
+def initialiseParticlesFromFile(cfg, avaDir, releaseScenario):
     # TODO: this is for development purposes, change or remove in the future
     # If initialisation from file
+
+    print('RELEASE ', releaseScenario)
```
Comment (fso42) [inline avaframe/com1DFA/particleTools.py:28]: Thi seems to be a debut print? If not, convert to log...

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/particleTools.py (hunk 3 above)
Comment (awirb) [reply avaframe/com1DFA/particleTools.py]: removed print

File: avaframe/com1DFA/com1DFA.py (hunk 2 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py:1910]: I added an atol of 1.e-6 as the file name where the time info is read if saved from com1DFA is set to %.2f so could even be less

**Review by matthiasto — APPROVED** [review]

**Review by fso42 — APPROVED** [review]

**Pull request merged**